.github/workflows/unittests.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
       fail-fast: false
       matrix:
         # Detail: `setup-envtest list`
-        kubernetes-version: ["1.28.3", "1.29.3", "1.30.0"]
+        kubernetes-version: ["1.28.3", "1.29.3", "1.30.0", "1.31.3"]
```

Comment:
1.28 is EOL so it's probably OK to remove it: https://kubernetes.io/releases/.

Reply:
I'm wondering if we should support 1.28 - 1.32 in the next training-operator release in this Dec.
Because the 1.28 is still actively supported in most popular cloud providers and the next release is final one for the v1 API.

But, after we release the 1.9.0 in Dec, I think that we should immediately remove the 1.28 related codes.

Reply:
@tenzen-y Don't we hit any limits in GitHub actions if we run our tests on 4 k8s version ?

Reply:
@andreyvelich AFAIK, there are no limits for open repos.